lib/identity_config.rb

```diff
@@ -420,6 +420,7 @@ def self.store
     config.add(:socure_docv_webhook_secret_key, type: :string)
     config.add(:socure_idplus_api_key, type: :string)
     config.add(:socure_idplus_base_url, type: :string)
+    config.add(:socure_idplus_shadow_mode_percent, type: :integer)
```

Comment:
I kind of wonder if some specialized syntax like `"1 / 1000"` would be good. This may come back around when we get to smooshing this A/B test in with the docv one a little more, but keeping this consistent is great for now.

Reply:
Just spitballing, if we did the `"1 / 1000"` thing I'd probably advocate for these configs ending in `_chance` rather than `_percent`, e.g.:

```yaml
socure_idplus_shadow_mode_chance: "1 / 1000"
```